### PR TITLE
Text corrections for company identities v1

### PR DESCRIPTION
**Pull request by concordium-cl**
## Purpose

I added some clarifications and corrections to the company identity flow v1.

## Changes

- aligned wording in both files (recovery-request.json)
- deleted that accounts cannot be recovered
- added last step to user and idp recovery description

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

**Comment by abizjak** [comment]
@concordium-cl @eb-concordium should this be merged, or is it waiting for something?

**Comment by concordium-cl** [comment]
@abizjak It's waiting for Emil's review. Guess he also wanted to add sth with checking time stamps.

File: rust-bins/docs/identity-provider-cli.md [hunk 1]
```diff
@@ -95,5 +95,7 @@ Upon receiving recovery request from the user, run
 ```console
 identity_provider_cli validate-recovery-request --cryptographic-parameters cryptographic-parameters.json \
                       --ip-info ip-info.json \
-                      --request request.json
-```
\ No newline at end of file
+                      --request recovery-request.json
+```
+
+Upon success the tool will display the ID recovery validation result. If it is `true`, return the user's identity object.
```
Comment (eb-concordium) [inline rust-bins/docs/identity-provider-cli.md:101]:
```suggestion
Upon success the tool will display the ID recovery validation result. The identity provider should return the identity object, if the following is satisfied:
-  the validation result it is `true`, and
-  the timestamp in the request does not deviate more than 15 minutes from the identity provider's current time,
-  the request is not a duplicate, i.e. the identity provider has not seen the very same request before. 
```

**Comment by eb-concordium** [comment]
Looks good. I only have one comment regarding the identity provider's acceptance of a recovery request.

File: rust-bins/docs/identity-provider-cli.md (hunk 1 above)
Comment (eb-concordium) [reply rust-bins/docs/identity-provider-cli.md:101]: I agree. I updated the suggestion.

**Review by eb-concordium — APPROVED** [review]

**Pull request merged**